include/GcInfo/Target.h

```diff
@@ -20,7 +20,46 @@
 
 #if (defined(_TARGET_X86_) || defined(_TARGET_X64_) || defined(_TARGET_AMD64_))
 
-// Define DWARF encodings for registers
+// Define encodings for machine registers
+
```

Comment:
Is this duplication of code that is in coreclr?  This looks an awful lot like a header that should be coming with the gcinfo lib.